content/02.introduction.md

```diff
@@ -22,29 +22,55 @@ As researchers worldwide work to characterize SARS-CoV-2 and COVID-19, informati
 In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
 This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.
 
-### Coronaviruses: What are they, and what do we know about SARS-CoV-19?
+### About Coronaviruses
 
-Coronaviruses are RNA viruses that...
-[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important]
+#### Classification:
+Coronaviridae, commonly called Coronavirus, is a family of positive-sense RNA viruses. Several members of this family include the Middle East respiratory syndrome-related coronavirus (MERS-CoV) and Severe acute respiratory syndrome coronavirus (SARS-CoV), which are two species.
 
+#### Virion, morphology, structure:
+The virion is spherical with a diameter ranging between 100 to 160nm. The virion has a lipid envelope with spike (S) glycoproteins on the surface creating the distinctive "crown" shape, from which the virus was named. 
+
+#### Genome structure and replication:
+The genome is a single-stranded non-segmented positive RNA (ssRNA+) that is 27 to 32kb in length. The genome is comprised of a replicase gene that is 2/3 of the genome and the remaining 1/3 that encodes structural proteins including: spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins. There are additional accessory genes that are present depending on the strain. The non-structural polypeptide is translated into 16 non-structural proteins (nsp1-16, except \gamma coronaviruses that do not contain nsp1), which form replication machinery to synthesize viral RNA. 
```

Comment:
I think this is info about coronaviruses generally and not SARS-CoV-2 in particular, but perhaps make that more explicit?

Reply:
We can cite this accession id and mention that the Wuhan Hu1 strain has 29kb (29,903 bp) genome. https://www.ncbi.nlm.nih.gov/nuccore/MN908947.

Reply:
Thats a very good point. Yes, the text was meant to give general CoV background. But I think adding was is known about SARS-CoV2 would be helpful.